src/server/controllers/image.controller.ts

```diff
@@ -220,6 +220,7 @@ export const getInfiniteImagesHandler = async ({
       user: ctx.user,
       headers: { src: 'getInfiniteImagesHandler' },
       include: [...input.include, 'tagIds'],
+      includeBaseModel: true,
```

Comment:
Not sure if it makes sense to `includeBaseModel` always here, maybe we can condition it if `input.baseModels.length > 0`